### DISCUSSION TITLE
NVIDIA Auto Frame Aspect Ratio [Bug?]

### DISCUSSION THREAD

**Discussion by SeffInSpace** [Xaymar/obs-stream-effects#981] [General]
Hi!

I just updated my OBS and plugins (including the NVIDIA APKs) and now the aspect ratio option in the Auto-Framing filter stretches the image immensely. I have been looking into fixing it through the internet and OBS itself and can't seem to find a fix. Is there one?

Thank you!

[Screenshot 2022-11-24 163431](https://user-images.githubusercontent.com/97858945/203878344-9e8c31a5-b1d4-415d-bd7a-488c942a2c2c.png)

**Comment by Xaymar** [comment]
Not much can be said without the accompanying log file from OBS Studio.

**Reply by SeffInSpace** [reply]
https://obsproject.com/logs/QUvTVl6DOUQpr7a2

Here you go, sorry!

**Reply by Xaymar** [reply]
```
17:01:14.640: OBS 28.1.2 (64-bit, windows)
17:01:16.842: [StreamFX] Loaded Version 0.12.0a151-g0f114f56
```

As you've probably already seen when downloading StreamFX v0.12, currently no support is given for those versions:

> ⚠ Preview Release not for Production! ⚠
> 
> This release is meant for testing in isolated conditions and early adopters only, and should never be used in production. If you use one of these releases, all promises of support, warranty, functionality, and similar are immediately void and if things break you are on your own. [The latest production ready release can be found here](https://github.com/Xaymar/obs-StreamFX/releases/latest).

This warning includes Discussions, Issues, Discord support, etc. For further support, downgrade to OBS Studio 27.2 with StreamFX 0.11.x again.

**Reply by SeffInSpace** [reply]
thanks, sorry for such a trivial solution 😭
